third_party/docfx/templates/devsite/UniversalReference.html.primary.js

```diff
@@ -13,6 +13,7 @@ exports.transform = function (model) {
   }
 
   model._disableToc = model._disableToc || !model._tocPath || (model._navPath === model._tocPath);
+  model._displayFriendlyApiName = model._displayFriendlyApiName && model.friendlyApiName

```

Comment:
Since we already have `{{#_displayFriendlyApiName}}` in the template, this is a redundant check.

Reply:
This checks that the ```friendlyApiName``` key in the yaml present and non-empty.

Reply:
Right, do we need to check that it's present and non-empty for a specific reason? Suppose that this JavaScript line isn't there, if `_displayFriendlyApiName` is not present then the template will not render it. That's why I think this is a redundant check, unless there's a reason I'm not aware of.

Reply:
There's a ```-``` character between the friendly api name and the existing title. If the ```friendlyApiName``` key is missing from the yaml, the dash character will still appear unless we have this additional condition.